src/main_loop.rs

```diff
@@ -1778,6 +1776,8 @@ mod test {
     }
 
     async fn setup(num_init_peers_outgoing: u8, num_peers_incoming: u8) -> TestSetup {
+        const CHANNEL_CAPACITY: usize = 10;
```

Comment:
Bad name. Maybe rename to `CHANNEL_CAPACITY_MINER_TO_MAIN`? I know you didn't make the name, but when you're moving it further from where it's being used, context is less clear than before.